test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -557,6 +557,15 @@ def test_direct_3ds_authorize
     assert auth.params['_links']['redirect']
   end
 
```

Comment:
I took `allow_upgrade` out of the 3ds2 options and wrote a dedicated remote test here as it seems it's intended for the gateway direct 3DS only. The field doesn't come back in the response so I can't add an assertion for anything other than the fact that the merchant is enrolled in 3DS, but happy to hear feedback on this test.